src/ng/templateRequest.js

```diff
@@ -36,7 +36,8 @@ function $TemplateRequestProvider() {
 
       var httpOptions = {
         cache: $templateCache,
-        transformResponse: transformResponse
+        transformResponse: transformResponse,
+        headers: { 'Accept': 'text/html, text/ng-template' }
```

Comment:
I have a feeling it would be safest to move this to the provider so that it could always be modified on an app by app basis.